e2e_playwright/st_columns.py

```diff
@@ -12,19 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 
-import streamlit as st
+import numpy as np
 
-# Construct test assets path relative to this script file to
-# allow its execution with different working directories.
-TEST_ASSETS_DIR = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "test_assets"
-)
+import streamlit as st
 
-CAT_IMAGE = os.path.join(TEST_ASSETS_DIR, "cat.jpg")
 LOREM_IPSUM = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
 
+BLACK_IMG = np.repeat(0, 601350).reshape(633, 950)
```

Comment:
Replaced the cat image with a black image since it seems that it sometimes leads to a non-sharp version of the image being screenshotted -> which likely is a result of some browser optimization that first shows a less detailed version of the image. Since the test is about columns and not our image rendering, I just replaced it with a black image to not run into this issue.